h/storage.py

```diff
@@ -24,7 +24,7 @@
 
 from h import models, schemas
 from h.db import types
-from h.util.group_scope import match as group_scope_match
+from h.util.group_scope import uri_in_scope
```

Comment:
Slightly confusingly this commit came _after_ the one that removed the _match_ function.

Reply:
Again with the https://help.github.com/en/articles/why-are-my-commits-in-the-wrong-order